openapi/endpoints/client-application/client-application-update-origins.yaml

```diff
@@ -0,0 +1,45 @@
+post:  
+  tags:
+    - client-application
+  summary: Update the allowed origins
+  description: |
+    Add url origins to the client application whitelist. 
+    The [Generate a challenge endpoint](/api-reference/generate-challenge) will reject non-whitelisted origins.
+    Note! This is a destructive operation. The allowed origins will be overwritten with the provided list.
```

Comment:
minor thing but to me an exclamation mark after `Note` looks odd. And probably it should be a stronger word if we want to bring people's attention to it. Maybe something like this?
`Warning: this is a destructive operation!`